presto-main/src/main/java/com/facebook/presto/operator/BigintGroupByHash.java

```diff
@@ -183,6 +183,12 @@ else if (value == values.get(hashPosition)) {
     }
 
     @Override
+    public boolean indeterminateContains(int position, Page page, int[] hashChannels)
```

Comment:
I will fix it. Also, I would also like to change the existing `contains` method to `containsExact`, otherwise it looks confusing.
@kokosing What do you think?